### PR TITLE
Add MockUniswap and MockMultiCollateral full implementation

### PR DESCRIPTION
**Pull request by kirilaa**

### CODE REVIEW

File: contracts/utils/test-helpers/MockUniswap.sol [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+// external
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./library/Path.sol";
```
Comment (danijelthales) [inline contracts/utils/test-helpers/MockUniswap.sol:7]: why is this required?

**Review by danijelthales — APPROVED** [review]

**Review by vladanthales — APPROVED** [review]
